### PR TITLE
added wrapper for SimConnect_FlightLoad

### PR DESCRIPTION
**Pull request by uriba107**
Added basic wrapper for SimConnect_FlightLoad.

in addition,
I've added a try catch for Win32 && x64 to check for manual built modules.

### CODE REVIEW

File: src/addon.cc [hunk 1]
```diff
@@ -671,6 +683,7 @@ void Initialize(v8::Local<v8::Object> exports) {
 	NODE_SET_METHOD(exports, "setAircraftInitialPosition", SetAircraftInitialPosition);
 	NODE_SET_METHOD(exports, "transmitClientEvent", TransmitClientEvent);
 	NODE_SET_METHOD(exports, "requestSystemState", RequestSystemState);
+	NODE_SET_METHOD(exports, "FlightLoad", FlightLoad);
```
Comment (EvenAR) [inline src/addon.cc]: I think it's better if all exported methods have the same casing in their names (`"FlightLoad"` -> `"flightLoad"`)

File: index.js [hunk 2]
```diff
@@ -11,7 +11,14 @@ if(process.platform === "win32" && process.arch === "ia32") {
...
+		// Try loading manual build first
+		nodeSimconnect = require('./build/Release/node-simconnect');
+		console.info("node-simconnect: Manual build loaded");
+	} catch(ex) {
+		console.error("node-simconnect: Node.js 64 bit is not supported. Please install Node.js 32 bit.")
+
+	}
 } else if(process.platform !== "win32") {
     console.error("node-simconnect: The only supported platform is Windows (win32). Found: " + process.platform)
 }
```
[the remark this answers is not in the document]
Comment (uriba107) [reply index.js]: Your are the coder around here, I'm just button mashing monkey :)

but it looks a lot cleaner to me.
I need to amend anyways, so I'll push it in as well

File: src/addon.cc (hunk 1 above)
Comment (uriba107) [reply src/addon.cc]: I'll fix the export line.
the function itself is like you had started, with function name identical to the sdk method it wrapped

**Comment by uriba107** [comment]
both changes commited.

**Comment by EvenAR** [comment]
Looks good! 👍 I will merge and publish the changes once I get time :) Really appreciate the contribution. Thank you!

**Pull request merged**